### PR TITLE
Update kramo.Cartridges to v1.5.5

### PR DESCRIPTION
**Pull request by kbdharun**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/109453&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/k/kramo/Cartridges/v1.5.5/kramo.Cartridges.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: kramo.Cartridges
+PackageVersion: v1.5.5
```
Comment (ItzLevvie) [inline manifests/k/kramo/Cartridges/v1.5.5/kramo.Cartridges.installer.yaml:5]:
```suggestion
PackageVersion: 1.5.5
```

File: manifests/k/kramo/Cartridges/v1.5.5/kramo.Cartridges.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: kramo.Cartridges
+PackageVersion: v1.5.5
```
Comment (ItzLevvie) [inline manifests/k/kramo/Cartridges/v1.5.5/kramo.Cartridges.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.5.5
```

File: manifests/k/kramo/Cartridges/v1.5.5/kramo.Cartridges.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: kramo.Cartridges
+PackageVersion: v1.5.5
```
Comment (ItzLevvie) [inline manifests/k/kramo/Cartridges/v1.5.5/kramo.Cartridges.yaml:5]:
```suggestion
PackageVersion: 1.5.5
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by kbdharun** [comment]
@ItzLevvie In all the previous versions of the manifest I have used `v` before the version similar to our GitHub tag, I have a doubt would existing manifest users be able to update seamlessly without unknown version errors (a definite upgrade path is critical as we recommend winget as the recommended way to get it in Windows). I will rename the directory to the suggested one, please clarify if there would be any disruption when updating before merging this PR.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**